apps/namada-interface/src/App/App.tsx

```diff
@@ -78,7 +90,8 @@ function App(): JSX.Element {
             <TopSection>
               <TopNavigation
                 isLightMode={isLightMode}
-                setIsLightMode={setIsLightMode}
+                toggleColorMode={toggleColorMode}
+                setColorMode={setColorMode}
```

Comment:
This one has to be passed, so we change theme to light when navigating to governance/staking.